01-Introducing-Molecular-Dynamics/02b-Running-Rigid-Body-Molecular-Dynamics-System.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>line 9 <code>position[:,0] = position[:,0]</code>  does nothing. In line 18 you index individually into <code>I_prin</code> just convert it to a tuple using <code>tuple</code>   or leave it as a list with <code>[I_prin] * N_particles</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>